tests/Util/Lifecycle/UpdateTest.php

```diff
@@ -412,6 +415,51 @@ public function testUpdateTo902(): void
         static::assertSame(ExperienceContext::LANDING_PAGE_TYPE_GUEST, $systemConfig->get(Settings::LANDING_PAGE, TestDefaults::SALES_CHANNEL, false));
     }
 
+    public function testUpdateTo1040(): void
```

Comment:
That test doesn't test anything, really. The new icons are already installed in the pipeline before the test. You could remove the media beforehand (or set to a different media entity).